plans/example-browser-node/browser/index.js

```diff
@@ -0,0 +1,105 @@
+const { chromium, firefox, webkit } = require('playwright')
+const { exit } = require('process')
+const { spawn } = require('child_process')
+
+const spawnServer = require('../server')
```

Comment:
Is the server only used to enable browser support?
Could we move the code inside the `browser` folder to make the split between browser support, node support, and actual test code, more obvious?

Reply:
Sure. it is used to serve the testground plan in the browser, but indeed, we can make it part of the `browser` folder, np!

Reply:
Moved `server` to `browser/server`: https://github.com/testground/testground/pull/1502/commits/01a187cf80f594a3eb8001da79734b28fae8b3fe